### ISSUE TITLE
 value of precision and recall are very low when training magicpoint on coco dataset

### ISSUE THREAD

**Issue by kebinC** [rpautrat/SuperPoint#20]
hello, 
i train magicpoint on coco dataset, but the value of precision and recall are very low

![image](https://user-images.githubusercontent.com/14308158/47360169-60933780-d701-11e8-8065-70d92c2d062f.png)

is it correct ? 

And i train the magicpoint using pretrained model trainded on synthetic_shapes dataset.

**Comment by Doooong** [comment]
I have the sample problems.It has been solved by @xz1cv. you can see the first issues ,it has the answer.

**Comment by rpautrat** [comment]
Yes exactly, thanks @Doooong. More precisely, the proposed solution was here: https://github.com/rpautrat/SuperPoint/issues/15#issuecomment-431005842 and I have updated the repo according to it now.

**Comment by kebinC** [comment]
when training synthetic data, the convergence of precison and recall are 0.4 and 0.5 correspondingly.
when training coco data, the convergence of precison and recall are 0.02 and 0.1 correspondingly.
